docs/source/benchmarking.rst

```diff
@@ -71,73 +90,84 @@ Aggregate across simulated communities (when dataset index is `not` unique)
     df = pd.concat(dfs)
     df.to_csv("benchmarks.tsv", sep='\t', index=True, header=True)
 
-Downloading Test Datasets
-=========================
-
-Using the built-in ``autometa`` module
---------------------------------------
-
-Autometa is packaged with a built-in module that allows any user to download any of the available test datasets.
-To use these utilities simply run the command ``autometa-download-dataset``.
+Benchmark classification
+^^^^^^^^^^^^^^^^^^^^^^^^
 
-For example, to download all of the simulated communities reference binning/taxonomy assignments as well as the Autometa
-v2.0 binning/taxonomy predictions:
+Previously we benchmarked the clustering step. Now we are going to benchmark the taxon profiling step. The only difference is that the ``--classification`` flag is provide instead of the ``--clustering`` flag.
```

Comment:
`--classification` and `--clustering` are values in the example below, not flags